Modules/Sources/Yosemite/Tools/POS/POSCatalogSyncCoordinator.swift

```diff
@@ -54,7 +62,21 @@ public actor POSCatalogSyncCoordinator: POSCatalogSyncCoordinatorProtocol {
 
         let catalog = try await fullSyncService.startFullSync(for: siteID)
 
-        settingsStore.setPOSLastFullSyncDate(Date(), for: siteID)
+        // Update the site with the full sync date
+        let currentSite = try await persistenceService.loadSite(siteID: siteID)
+        let updatedSite = POSSite(
+            siteID: siteID,
+            lastIncrementalSyncDate: currentSite?.lastIncrementalSyncDate,
+            lastFullSyncDate: Date()
+        )
```

Comment:
nit: how about the following in case the site properties grow?

```suggestion
        let updatedSite = currentSite.toPOSSite().copy(lastFullSyncDate: Date())
```